code/modules/projectiles/gun.dm

```diff
@@ -945,6 +950,45 @@
 		flash_loc.vis_contents -= muzzle_flash
 	muzzle_flash.applied = FALSE
 
+// for guns firing on their own without a user
+/obj/item/gun/proc/discharge(cause, seek_chance = 50)
+	var/target
+	if(!safety)
+		// someone is very unlucky and about to be shot
+		if(prob(seek_chance))
```

Comment:
This needs to be VERY VERY VERY low or not exist at all.

Reply:
base chance has been reduced to 10%, and on throw now has been reduced to a 50% chance, as a coin flip hail mary option to throw your gun as a last resort

Reply:
Still think it's too much honestly, should be pretty rare and absolutely not reliable

Reply:
5 and 20 sound okay for base and throw respectively?

Reply:
Chance to seek is just a flat 10% now, and should be seen even less with the reduced misfire chances now.